stories/SQFormAsyncAutocomplete.stories.js

```diff
@@ -34,41 +35,38 @@ const defaultArgs = {
   children: [
     {
       label: 'First',
-      value: 'first'
+      value: 'first',
     },
     {
       label: 'Second',
-      value: 'second'
+      value: 'second',
     },
     {
       label: 'Third',
-      value: 'third'
+      value: 'third',
     },
     {
       label: 'Fourth',
-      value: 'fourth'
+      value: 'fourth',
     },
     {
       label: 'Fifth',
-      value: 'fifth'
-    }
+      value: 'fifth',
+    },
   ],
-  handleAsyncInputChange: action('handleAsyncInputChange')
+  handleAsyncInputChange: action('handleAsyncInputChange'),
 };
 
-const Template = args => {
-  const {schema, SQFormProps, ...rest} = args;
+const Template = (args) => {
+  const {schema, SQFormProps, size, ...rest} = args;
   return (
     <div style={{minWidth: 250}}>
       <SQFormStoryWrapper
         initialValues={{[defaultArgs.name]: ''}}
         validationSchema={schema}
         {...SQFormProps}
       >
-        <SQFormAsyncAutocomplete
-          {...rest}
-          size={args.size !== 'auto' ? Number(args.size) : args.size}
-        />
+        <SQFormAsyncAutocomplete {...rest} size={getSizeProp(size)} />
```

Comment:
`getSizeProp` was used in any story that had the size prop customized via storybook controls. This fixes console warnings.